src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/custom.py

```diff
@@ -118,11 +119,14 @@ def create(cmd, client, resource_group_name, resource_name, kind, description=No
         logger.info('Bot parameters client side validation successful.')
         logger.info('Creating bot.')
 
-        return client.bots.create(
+        result = client.bots.create(
             resource_group_name=resource_group_name,
             resource_name=resource_name,
             parameters=parameters
-        )
+        ).as_dict()
+        if show_password:
+            result['password'] = password
```

Comment:
you do understand that the `password` could be None, and is the command fine with that?

Reply:
I've changed the show_password value to be set to True during the call to create a Converged App.

This way if a password is created on behalf of the user, the user will know what the password is. (Fixes: Microsoft/botbuilder-tools#1021)

If the password is already provided, the user has the password and doesn't need the information again.

The password can only be None if the user provides an appid, in which case they can access the portal(s) or use `az ad app update` to create a password.

Thanks for catching that!